modules/apps/portal/portal-inactive-request-handler/src/main/resources/META-INF/resources/inactive.jsp

```diff
@@ -0,0 +1,25 @@
+<%--
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+--%>
+
+<%@ include file="/init.jsp" %>
+
+<aui:style>
```

Comment:
This is OK because the tag will take care of adding a nonce when needed and since the code is no longer inside an inline `style` attribute CSP won't complain.